massa-db-worker/src/massa_db.rs

```diff
@@ -837,3 +850,803 @@ impl MassaDBController for RawMassaDB<Slot, SlotSerializer, SlotDeserializer> {
         self.recompute_db_hash()
```

Comment:
IIRC, recompute_db_hash() should not be used in the codebase (this statement should be checked).
It was introducted when computing the hash took too much time with LSM trees.

Could you check that, and either remove it (in this PR or another) or explain in comments?

Reply:
I would do it in another PR so we keep this PR focused on unit tests :)